### ISSUE TITLE
data attributes in Keynote::Rumble

### ISSUE THREAD

**Issue by igas** [rf-/keynote#13]
Hi @rf- 

I noticed recently by extracting some rails helpers or haml chunks of code that I can not use something like:

```ruby
build_html do
  a("Link", "#", data: { modal: true })
end
```

will generate:

```html
<a href="#" data="{:modal=>true}">Link</a>
```

While rails `link_to("Link", "#", data: { modal: true })`:

```html
<a data-modal="true" href="#">Link</a>
```

And haml `%a{ href: "#", data: { modal: true } } Link`:

```html
<a data-modal href="#">Link</a>
```

Do you think keynote should have support for this `data` attributes?

**Comment by rf-** [comment]
Great point, thanks! I'll release a new version once the build runs.

**Comment by rf-** [comment]
Just released v1.0.0 with this change 👌

**Comment by igas** [comment]
Thanks @rf- ! Array and aria support looks awesome too! 👍
